packages/apps/dashboard/server/src/modules/redis/redis.service.ts

```diff
@@ -0,0 +1,29 @@
+import { Inject, Injectable, Logger } from '@nestjs/common';
+import { Cache, CACHE_MANAGER } from '@nestjs/cache-manager';
+
+import { RedisConfigService } from '../../common/config/redis-config.service';
+
+@Injectable()
+export class RedisService {
+  private readonly logger = new Logger(RedisService.name);
+  constructor(
+    @Inject(CACHE_MANAGER) private cacheManager: Cache,
+    private readonly redisConfigService: RedisConfigService,
+  ) {}
+
+  async getHmtPrice(): Promise<number> {
+    const cachedHmtPrice: number | undefined = await this.cacheManager.get(
+      this.redisConfigService.hmtPriceCacheKey,
+    );
+    return cachedHmtPrice;
+  }
+
+  async setHmtPrice(hmtPrice: number): Promise<void> {
+    this.logger.log(`Setting a new HMT price: ${hmtPrice}`);
```

Comment:
makes sense to log this for every update?
Maybe would make sense to save on Redis also the last date it was updated so that we can check when was the last update to make sure it is working fine.